src/SignalR/common/Protocols.MessagePack/src/Protocol/MessagePackHubProtocol.cs

```diff
@@ -656,17 +627,17 @@ private static object DeserializeObject(ref MessagePackReader reader, Type type,
             }
         }
 
-        internal static List<IFormatterResolver> CreateDefaultFormatterResolvers()
-        {
-            // Copy to allow users to add/remove resolvers without changing the static SignalRResolver list
-            return new List<IFormatterResolver>(SignalRResolver.Resolvers);
-        }
+        internal static MessagePackSerializerOptions CreateDefaultFormatterResolvers() =>
```

Comment:
Rename `CreateDefaultFormatterResolvers` since it does more than just resolvers now.

Reply:
on it